oval_graph/command_line_client/client.py

```diff
@@ -204,7 +204,8 @@ def prepare_parser(self, parser):
             '--output',
             action="store",
             default=None,
-            help='The file where to save output.')
+            help=("The path where to save the output. If there is more report"
+                  " to generate it will create an OUTPUT path as a directory."))
```

Comment:
```suggestion
            help=("The path where to save the output. If there are more report "
                  "files to generate it will create an OUTPUT path as a directory."))
```